src/react/components/common/tagInput/tagInputToolbar.tsx

```diff
@@ -116,14 +125,25 @@ export default class TagInputToolbar extends React.Component<ITagInputToolbarPro
                     );
                 } else if (itemConfig.category === Categories.Separator) {
                     return (<div className="tag-input-toolbar-separator" key={itemConfig.displayName}></div>);
-                } else if (itemConfig.category === Categories.Modifier) {
+                } else if (itemConfig.category === Categories.RenameModifier) {
+                    return (
+                        <IconButton
+                            key={itemConfig.displayName}
+                            disabled={renameModifierDisabled}
+                            title={itemConfig.displayName}
+                            ariaLabel={itemConfig.displayName}
+                            className={renameModifierClassName}
+                            iconProps={{iconName: itemConfig.icon}}
+                            onClick={(e) => this.onToolbarItemClick(e, itemConfig)} />
+                    );
+                } else if (itemConfig.category === Categories.MoveModifier) {
                     return (
                         <IconButton
                             key={itemConfig.displayName}
-                            disabled={modifierDisabled}
+                            disabled={moveModifierDisabled}
                             title={itemConfig.displayName}
                             ariaLabel={itemConfig.displayName}
-                            className={modifierClassName}
+                            className={moveModifierClassName}
```

Comment:
We could use, instead of all above with class names, ternary and string literals like:
```suggestion
                            className={`tag-input-toolbar-iconbutton-disabled ${ moveModifierDisabled? "tag-input-toolbar-iconbutton-disabled" : " " }`}
```